### ISSUE TITLE
Add chart to Helm Hub

### ISSUE THREAD

**Issue by NorDroN** [aledbf/kube-keepalived-vip#106]
Hi!
What about to add this chart to [Helm Hub](https://hub.helm.sh/charts)?
More info [here](https://github.com/helm/hub/blob/master/Repositories.md).

**Comment by steven-sheehy** [comment]
Yes, that was the plan, but first you have to host the chart somewhere. That's why I added support to host it via GitHub pages in my [PR](https://github.com/aledbf/kube-keepalived-vip/pull/94), but it hasn't been reviewed in 5 months.

**Comment by NorDroN** [comment]
Any updates?
